libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_api.c

```diff
@@ -183,6 +203,10 @@ static void _incomingPingresp( void * pArgument )
     static int32_t invokeCount = 0;
     static uint64_t lastInvokeTime = 0;
     uint64_t currentTime = IotClock_GetTimeMs();
+    char pKeepAliveStatus[ KEEP_ALIVE_PERIODIC_STATUS_LENGTH ] = { 0 };
```

Comment:
This seems like a large buffer to create on stack. I see that there are other static variables in the function which means that function is not reentrant. Does it make sense to make this buffer static as well?

Reply:
I didn't think 128 bytes was that large. We can make it static

Reply:
That depends on the value of stack size. If the size of stack is 1KB, then 128 bytes is significant. And in case of Nuvoton, the size of stack is actually 1 KB: https://github.com/aws/amazon-freertos/blob/master/vendors/nuvoton/boards/numaker_iot_m487_wifi/aws_tests/config_files/iot_config.h#L48

Reply:
thanks